spec/basic/15_arithmetic_and_lists/input.scss

```diff
@@ -29,4 +29,6 @@ div {
   r: 16em * 4;
   s: (5em / 2);
   t: 1 + (2 + (3/4 + (4/5 6/7)));
+  u: (1 / 0); // Infinity
+  v: (0 / 0); // NaN
 }
```

Comment:
I don't understand why these test cases were added to a directory called "arithmetic and lists" when they don't seem to have anything to do with lists.